app/lib/esQueryBuilder.js

```diff
@@ -5,13 +5,73 @@ function build(searchTerm) {
     body: {
       size: 30,
       query: {
-        multi_match: {
-          query: searchTerm,
-          fields: ['name^2', 'address.addressLines', 'doctors'],
-          type: 'cross_fields',
-          operator: 'and'
+        bool: {
+          should: [
+            {
+              match_phrase: {
+                name: {
+                  query: searchTerm,
+                  boost: 2,
+                  slop: 1
```

Comment:
'Slop' allows a words to be missing or transposed in a phrase. The value is for the count of words which are allowed to be sloppy. It seems to give good results when, for example, searching for '<something> Medical Centre' and it's actual name is '<something> Health Centre'.